packages/p2p/src/utils/string.js

```diff
@@ -10,6 +10,10 @@ export const countDecimalPlaces = value => {
 };
 
 export const generateHexColourFromNickname = nickname => {
+    if (!nickname) {
+        return '??';
```

Comment:
Why `??` and not an empty string `''`?

Reply:
This situation will almost never occur, but if it does we can always expect the function to return two characters.